src/state/queries/post-thread.ts

```diff
@@ -329,8 +330,10 @@ function responseToThreadNodes(
 ): ThreadNode {
   if (
     AppBskyFeedDefs.isThreadViewPost(node) &&
-    AppBskyFeedPost.isRecord(node.post.record) &&
-    AppBskyFeedPost.validateRecord(node.post.record).success
+    atp.fastIsType<AppBskyFeedPost.Record>(
+      node.post.record,
+      AppBskyFeedPost.isRecord,
+    )
```

Comment:
This effectively removes full record validation from this check. Double checking that we don't need this [here in Slack.](https://bluesky-builders.slack.com/archives/C04CY4895NK/p1736207759085869)

Reply:
Yep we're good 👍